public/nav-logo.svg

```diff
@@ -0,0 +1,32 @@
+<svg width="172" height="29" viewBox="0 0 172 29" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Did you make this SVG file to go through https://jakearchibald.github.io/svgomg/?

Reply:
@marbiano this svg is directly getted of figma, i should use that website? what is the difference?